packages/riipen-ui/src/components/Paper.jsx

```diff
@@ -0,0 +1,111 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+import _JSXStyle from "styled-jsx/style";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+const Paper = props => {
+  const {
+    children,
+    classes,
+    component: Component,
+    rounded,
+    elevation,
+    variant,
+    ...other
+  } = props;
+
+  const theme = React.useContext(ThemeContext);
+
+  const className=clsx(
+    "root",
+    classes,
+    rounded ? "rounded" : null,
+    variant === "outlined" ? "outlined" : "elevation"
+  );
+  
+  return (
+    <React.Fragment>
+    <Component 
+    className={className}
+      {...other}
+    >
+      {children}
+      </Component>
+     
+      <style jsx>
+        {`
+          .root {
+            background-color: ${theme.palette.common.white};
+            color: ${theme.palette.text.primary}
```

Comment:
I think you can actually remove this. This would define any colour for text inside the component, but we don't actually care what someone renders inside the paper, that should be left to them.